webpack/configs/base.ts

```diff
@@ -0,0 +1,36 @@
+import * as webpack from "webpack";
+
+import { TsConfigPathsPlugin } from "awesome-typescript-loader";
+
+const CommongConfig: webpack.Configuration = {
+
+    entry: "./webpack/globalize.ts",
+
+    output: {
+        path: `${__dirname}/../../build`
```

Comment:
This relative path is still a bit messy.

Reply:
It's actually an absolute path to "build" from the file system root. The problem is that there is no system variable that links into the project root, but only the __dirname, which points to the location of the config file, so we have to trace back a couple folders manually to get to "build".

---
In reply to: [250238812](https://github.com/Microsoft/clarity-js/pull/126#discussion_r250238812) [](ancestors = 250238812)